### PR TITLE
add shell/powershell script to root dir for easy JRE generation

### PR DESCRIPTION
**Pull request by gdams**

### CODE REVIEW

File: sbin/build.sh [hunk 1]
```diff
@@ -988,6 +988,28 @@ createNoticeFile() {
   fi
 }
 
+# Adds a script for easy JRE generation using jlink
+addCreateJREScript() {
```
Comment (karianna) [inline sbin/build.sh:992]: Let's not call it a JRE as that's an official designation.  I'd change it to Runtime or something like that

**Review by karianna — CHANGES_REQUESTED** [review]

**Review by sxa — COMMENTED** [review]
We do need to carefully look at the comments/feedback we've had in https://github.com/adoptium/temurin-build/issues/2683 and the [slack](https://adoptium.slack.com/archives/C09NLQQAV/p1632733549158800) [conversations](https://adoptium.slack.com/archives/C09NLQQAV/p1632955929226600) to see if this would alleviate people's objections. There has obviously been some resistance to just running the one liner we have provided them with. Would an extra script that (fundamentally) just runs that one-liner really alleviate those concerns for most people (it doesn't for the download size issue, but likely does for the bundling scneario) and means we're basically shipping an Adoptium specific tool with the JDK which is a new precedent to be setting, so we do need to be careful about this decision and it's perception given how we claim to be a generally "pure" OpenJDK. We did in the past come up with a name for a "JRE alternative" too which we could use for this.

File: sbin/createJRE.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    --no-man-pages \
+    --no-header-files \
+    --compress=2 \
+    --output "$jreDirectory"
+
+echo "Testing generated JRE"
+"$jreDirectory/bin/java" --version
+echo -e "Java Version test passed ✅\n"
+
+echo "Success: You're JRE runtime is available at $jreDirectory"
```
Comment (sxa) [inline sbin/createJRE.sh:51]:
```suggestion
echo "Success: Your JRE runtime is available at $jreDirectory"
```

File: sbin/createJRE.sh [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    --strip-debug \
+    --strip-debug \
+    --no-man-pages \
+    --no-header-files \
+    --compress=2 \
+    --output "$jreDirectory"
+
+echo "Testing generated JRE"
+"$jreDirectory/bin/java" --version
+echo -e "Java Version test passed ✅\n"
```
Comment (sxa) [inline sbin/createJRE.sh:49]: This really needs to check the return code from the java command, as it displays the message even when it fails. Also same comment for `jlink`

**Review by sxa — CHANGES_REQUESTED** [review]

File: sbin/createJRE.sh [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    echo "e.g ./makeJRE.sh /home/user/jreruntime"
+    exit 1
+fi
+
+jreDirectory="$1"
+binPath="$jdkDirectory/bin"
+
+"$binPath/jlink" --add-modules ALL-MODULE-PATH \
+    --strip-debug \
+    --strip-debug \
```
Comment (sxa) [inline sbin/createJRE.sh:41]: Duplicate line?

File: sbin/createJRE.sh (hunk 3 above)
Comment (gdams) [reply sbin/createJRE.sh]: added some

**Review by sxa — DISMISSED** [review]

**Pull request closed without merging**